### PR TITLE
CB-13543: fixing check for travis master branch

### PR DESCRIPTION
**Pull request by audreyso**
### What does this PR do?
fixing check for travis master branch

### Checklist
- [X] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [X] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [X] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by audreyso** [comment]
The only thing I wasn't sure about is if we need `#!/bin/bash` at the top of the travis.yml file. I think we only need it in the .sh file. https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps

File: .travis.yml [hunk 1]
```diff
@@ -7,17 +7,5 @@ before_install:
...
-    svn add --force .;
-    svn --non-interactive --no-auth-cache --username=$svn_username --password=$svn_pass commit -m "Updated docs";
-    cd ../cordova-docs;
-    npm test;
-  else
-    npm test;
-  fi'
\ No newline at end of file
+  - chmod 755 buildAndDeploy.sh
+  - travis_wait 120 sleep infinity & ./buildAndDeploy.sh
```
Comment (stevengill) [inline .travis.yml]: might want to change the 120 to 30 i think

Comment (audreyso) [reply .travis.yml]: fixed!

**Comment by audreyso** [comment]
Thanks! going to merge

**Pull request merged**